src/web/app/pages-student/student-home-page/student-home-page.component.scss

```diff
@@ -1,3 +1,8 @@
 .margin-0 {
   margin: 0;
 }
+
+.td-text-break {
```

Comment:
Could we use bootstrap's `text-break` instead?

Reply:
There's a [bug](https://github.com/twbs/bootstrap/issues/30803) with `text-break` in the current version of bootstrap css file used (4.5.0) that causes it to not work with certain tags, `<td>` being one of them.

v 4.5.0
<img width="1233" alt="Screenshot 2021-07-19 at 9 15 53 PM" src="https://user-images.githubusercontent.com/60355570/126165585-cccd0e28-2743-4001-98f0-52f21c9bdff4.png">

v 4.4.1
<img width="931" alt="Screenshot 2021-07-19 at 8 25 10 PM" src="https://user-images.githubusercontent.com/60355570/126165621-ae16b707-4a8b-45d5-a284-f495d3578bbf.png">

I used the `td-text-break` class to work around this. If this is not suitable maybe I could wrap all the necessary tags with a span and apply the `text-break` class to the span instead.

Reply:
I think the approach of using `td-text-break` is fine. Just put it in the global `styles.scss` to avoid repeating it everywhere, and leave a comment there to explain the rationale for this class.

> Actually would it be possible to look into changing the version of bootstrap stylesheet used?

What version do you want to change to? Is this issue fixed in a later version of bootstrap? We could create a separate issue for it since there may be breaking changes in changing the version. For this PR, I think we best stick to the current bootstrap version.

Reply:
Changing it is possible but unlikely within the context of one PR, but if many bugs require a change or we do a major patch we can make the change. 

For now, what about moving .td-text-break to a common scss file (such as `page.component`). This way we can also centralize the class name so we can also file an issue/ add to wiki to remove it when we update bootstrap.